maxun-core/src/utils/worker.ts

```
@@ -0,0 +1,157 @@
+                        url: pageUrl,
+                        error: error.message
+                    }
+                });
+                continue;
+            }
+        }
+
+        return results;
+
+    } catch (error) {
+        throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the catch clause.


(lint/complexity/noUselessCatch)